build/repo.props

```diff
@@ -0,0 +1,19 @@
+﻿<Project>
+  <PropertyGroup>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <DotNetCoreRuntime Include="2.0.0" />
+    <DotNetCoreRuntime Include="2.1.0-preview1-26016-05" />
+  </ItemGroup>
+
+  <Target Name="GetBuildNumber">
```

Comment:
This isn't called by default. It's a repo-specific target people *can* use.

I find this target incredibly useful because I often install WebHooks packages locally and use them in other projects. It also helps to work around issues like https://github.com/NuGet/Home/issues/6379 and create a self-consistent set of packages.

The pattern is:
```
> git clean -dfx
...
> .\build.cmd /t:GetBuildNumber /nologo
Using KoreBuild 2.1.0-preview1-15678
Adding C:\Users\dougbu\.dotnet\x64 to PATH
.NET Core runtime 2.0.5 is already installed. Skipping installation.
.NET Core SDK 2.2.0-preview1-007947 is already installed. Skipping installation.
>>> dotnet.exe msbuild @C:\dd\dnx\Universe++\WebHooks\artifacts\logs\msbuild.rsp
  /p:BuildNumber=t005c1c2fc

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:01.02
> .\build.cmd /p:BuildNumber=t005c1c2fc /nologo
...
```

Reply:
`repo.targets` then.